sdk/servicebus/service-bus/src/receiver.ts

```diff
@@ -108,6 +109,10 @@ export class Receiver {
    * amount of time to wait for a new message before closing the receiver.
    *
    * @returns void
+   * @throws {Error} Thrown if the underlying connection or receiver is closed.
```

Comment:
We don't explicitly catch and throw error when a client is closed in this code path, so it may be misleading? Could be related to this bug as well - https://github.com/Azure/azure-sdk-for-js/issues/5317

Reply:
The sender and the receiver both use same patterns to determine whether or not error should be thrown when connection or sender/receiver or client is closed.

Since you documented the case of closed client resulting in an error for sender, I assumed the same would apply to the receiver.

Reply:
Ok, yes the error message construction is same - updated

Reply:
But like you said, we don't explicitly throw error for when client is closed. Can you test this scenario and confirm if we throw error or not?
- create queueclient
- create receiver
- close queuecleint
- call `receiveMessages` on the receiver

Reply:
We do as part of the error message construction. About the bug in implementation - that can be investigated as part of #5317